aead/src/lib.rs

```diff
@@ -0,0 +1,121 @@
+//! A set of traits designed to support authenticated encryption.
+
+#![no_std]
+
+#[cfg(feature = "alloc")]
```

Comment:
Since we don't have non-allocating methods right now we don't need the `alloc` feature, so you can remove it completely.

Reply:
OK